engine/src/main/scala/cromwell/webservice/SwaggerUiHttpService.scala

```diff
@@ -94,6 +96,11 @@ trait SwaggerUiConfigHttpService extends SwaggerUiHttpService {
  * swagger UI, but defaults to "yaml". This is an alternative to spray-swagger's SwaggerHttpService.
  */
 trait SwaggerResourceHttpService {
+
+  def getBasePathOverride(): Option[String] ={
```

Comment:
Did you look into if there is a way to mock environment variables in the unittest? In Java I think there is something like "withEnvironmentVariables", but I don't know what is available in the Scala tests.

Reply:
I've looked and haven't found anything.